src/_guides/language/analysis-options.md

```diff
@@ -407,6 +419,18 @@ To suppress more than one rule, supply a comma-separated list:
 const x = y;
 ```
 
+To suppress all linter rules, add a `type=lint` specifier:
```

Comment:
I personally would probably not document in this position; The `type=lint` support use case was really just for `// ignore_for_file`, but of course it does work with `// ignore:` for consistency. I think this might just be unnecessary text here. Just my 2c.